src/pages/home/Home.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import styles from './home.scss';
+
+const logo = require('assets/logo.png');
+
+const Home = () => {
+  return (
+    <div className={styles.homeContainer}>
+      <div className={styles.mainHeaderBackground} />
+      <nav className={styles.navigationBar}>
+        <div className={styles.leftNavLinks}>
+          <a href="#" className={styles.navLinks}>
+            About
+          </a>
+          <a href="#" className={styles.navLinks}>
+            Mission
+          </a>
+          <a href="#" className={styles.navLinks}>
+            Contact
+          </a>
+        </div>
+      </nav>
+      <div className={styles.mainHeaderContent}>
+        <div>
+          <h1 className={styles.header}>Coding</h1>
+          <h1 className={styles.header}>Coach</h1>
+          <h4>Connecting developers with mentors worldwide</h4>
+          <button className={styles.learnMore}>Learn More</button>
+        </div>
+        <div>
+          <img src={logo} />
```

Comment:
I think we should use alt attributes for all images for web accessibility, so that visually impaired users using screen readers will be able read an alt attribute to better understand an on-page image and also it provide better image context/descriptions to search engine crawlers, helping them to index an image properly.

Reply:
i.e. `aria-label="coding coach logo"` and `role="img"`

Reply:
Yes alt images are non-negotiable!

Reply:
You don't need `role="img"` because it's an img tag so it's already semantic.

Reply:
This has been solved.